### DISCUSSION TITLE
How to manage utterance comments when closed?

### DISCUSSION THREAD

**Discussion by soerenthomsen** [OceanGlidersCommunity/Oxygen_SOP#106] [General]
The community comment function works well. Thanks to @utterance

I have not found a good way to get rid of closed comments beside erasing them when implemented. 
i.e. if i close the whole issues for example new comments on this section also get easily lost as they are shown under closed issues. 

@jbusecke any ideas? 

One idea is that the maintainers transform all comments into issues and remove the comments afterwards? 
This might make sense as often similar issues already exist? Typos / formulations can be implemented directly

For sure @tomhull @patricialg and myself have to agree on an approach to be organized for the community review.

**Comment by jbusecke** [comment]
I think that the utterance comments are meant to stay open. One possible solution is to keep them there until the public review phase is over and then just remove that functionality for the final website? I belive the github issues will stay the same in that case.

**Reply by soerenthomsen** [reply]
Yes maybe leaving them open is a good idea. But if they are dealt with in issues it is key to edit them and link people there to have an organised discussion and not having the same kind of comments etc. 

You can see how I did this with Pierre's recent comments. 

In general it is much less work if people start classical issues but the utterance comment style helps people who are new on GitHub for sure.

**Reply by jbusecke** [reply]
Yeah I see that. How about we leave it as is and wait for e.g. a week and collect some feedback on how hard it is to maintain (for you guys) and how people find the experience?

**Reply by soerenthomsen** [reply]
Yes fine for me. My aim is to have an as organised discussion as possible. So long commentaries below each sections is not really well organised. Thus I am in favour to close them after they incorporated and/or tranferred into issues. 

You can see how I did this with issue #114 for example.

**Reply by soerenthomsen** [reply]
This is also as we will need to come to a decision process as described in the contributions guide. This is only possible with issues and reviews etc.
